homeassistant/components/jewish_calendar/binary_sensor.py

```diff
@@ -0,0 +1,66 @@
+"""Support for Jewish Calendar binary sensors."""
+import logging
+
+from homeassistant.components.binary_sensor import BinarySensorDevice
+import homeassistant.util.dt as dt_util
+
+from . import DOMAIN, SENSOR_TYPES
+
+import hdate
```

Comment:
Put this as 3rd party import group between standard library and homeassistant import groups.